main/glossary/README.md

```diff
@@ -44,42 +54,84 @@ In Agoric documentation, *contract* usually refers to a contract's source code t
+
+It uses [object capabilities](#OCAP) to enforce access control. Instead of having 
+to prove ownership of a corresponding private key, if your program has a 
+reference to an object, it can call methods on that object. If it doesn't 
+have a reference, it can't. For more on object capabilities, see [Chip Morningstar's
 post](http://habitatchronicles.com/2017/05/what-are-capabilities/).
 
+Key ERTP concepts include [Issuers](#Issuers), [Mints](#Mints), 
+[Purses](#Purses), [Payments](#Payments), [Brands](#Brands), and [Amounts](#Amounts). Also 
+see the [ERTP Introduction](https://agoric.com/documentation/getting-started/ertp-introduction.html)
+and [ERTP Guide](https://agoric.com/documentation/ertp/guide/).
+
+<a name="Extents"></a>
 ## Extent
-Extents describe the extent of something that can be owned or shared: How much, how many, or description of unique asset. (Pixel(3,2), $3 or ‘Right to occupy on Tuesdays’). Fungible extents are normally represented by natural numbers. Other extents may be represented as strings naming a particular right, or an arbitrary object that sensibly represents the rights at issue.
+Extents are the part of an [amount](#Amounts) that describe the extent of something
+that can be owned or shared: How much, how many, or a description of a unique asset, such as
+Pixel(3,2), $3 or ‘Right to occupy on Tuesdays’. [Fungible](#Fungible) extents are usually 
+represented by natural numbers. Other extents may be represented as strings naming a particular
+right, or an arbitrary object representing the rights at issue. The latter two examples 
+are usually [non-fungible](#Non-fungible) assets. Extents must be [Comparable](#Comparable).
+
+See the [ERTP Guide's Extent section](https://agoric.com/documentation/ertp/guide/extent.html) for more information.
 
-Extent must be Comparable.
+<a name="Fungible"></a>
+## Fungible
+A fungible asset is one where all exemplars of the asset are interchangable. For example, if you 
+have 100 one dollar bills and need to pay someone five dollars, it does not matter which
+five one dollar bills you use. Also see [non-fungible](#Non-fungible).
 
 ## Handle
 A handle is a unique identifier implemented as a JavaScript object. Only its identity is meaningful, so handles do not have properties. Unlike number or string identifiers, handles are unforgeable. This means the only way to know a handle identity is being given an object reference, and no identity can be guessed and no fake identity will succeed. 
 
 For example, Zoe often uses `offerHandle` to refer to offers. Zoe contracts can use an offer's `offerHandle` as the key for requesting the current allocation of an offer or reallocating the offer's assets.
-
+<a name="Issuers"></a>
```

Comment:
It would be nice to maintain the blank line between sections. (It won't change the displayed text, but it's sometimes easier to edit that way.